pkg/agents/team5/messages.go

```diff
@@ -206,3 +206,91 @@ func (a *CustomAgent5) HandleStateIntendedFoodTaken(msg messages.StateIntendedFo
 	a.updateIntentionFoodMemory(msg.SenderID(), statement)
 	//a.Log("New value of intendedFood", infra.Fields{"statement": statement, "memory": a.socialMemory[msg.SenderID()].intentionFood})
 }
+
+func (a *CustomAgent5) RejectTreaty(msg messages.ProposeTreatyMessage) {
+	reply := msg.Reply(a.ID(), a.Floor(), msg.SenderFloor(), false)
+	a.SendMessage(reply)
+	a.Log("Rejected treaty", infra.Fields{"proposerID": msg.SenderID(), "proposerFloor": msg.SenderFloor(), "treatyID": msg.TreatyID()})
+}
+
+func (a *CustomAgent5) HandleProposeTreaty(msg messages.ProposeTreatyMessage) {
+	treaty := msg.Treaty()
+	if len(a.ActiveTreaties()) > 0 || treaty.Request() == messages.Inform {
+		//TODO: Have a checkForConflicts function that determines if we can feasibly sign up to multiple treaties.
+		//For now, only agree to sign up to 1 treaty at a time for simplicity
+		//Also, reject any inform treaties as I'm not sure what they are or how to handle them
+		a.RejectTreaty(msg)
+		return
+	}
+
+	if (treaty.Request() == messages.LeaveAmountFood && treaty.RequestOp() == messages.EQ) || treaty.RequestOp() == messages.LE || treaty.RequestOp() == messages.LT {
```

Comment:
might be wrong, but aren't the brackets in the wrong place here? shouldn't it be:
`if treaty.Request() == messages.LeaveAmountFood && (treaty.RequestOp() == messages.EQ || treaty.RequestOp() == messages.LE || treaty.RequestOp() == messages.LT) {`
or (slightly) more simply:
`if treaty.Request() == messages.LeaveAmountFood && treaty.RequestOp() != messages.GT && treaty.RequestOp != messages.GE {`

Reply:
My thinking was this: 
Code is meant to reject all treaties with less than (LE or LT) requests, but only reject equal to (EQ) requests if the request type is for a specific food amount. We would still accept treaties for equal to (EQ) requests if the request type is for a percentage amount of food.

Reply:
ah ok, yes I guess the only operations are to leave food so this makes sense!